SignalMessaging/attachments/SharingThreadPickerViewController.m

```diff
@@ -149,6 +149,12 @@ - (void)threadWasSelected:(TSThread *)thread
 
     NSString *_Nullable messageText = [self convertAttachmentToMessageTextIfPossible];
 
+    // Hide the navigation bar before presenting the approval view.
+    //
+    // Note that cancelling in the approval views will dismiss the entire
+    // share extension, so there is no "back" button.
+    self.navigationController.navigationBarHidden = YES;
```

Comment:
Previously we hid the navigation bar throughout the SAE, but we need to be able to "back" from the "find by phone number" view and its "country code" view.

Now, we hide the navigation bar only during the last step, the approval views.